src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
-public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+import java.util.Random;
+
+class ColorSupplier {
+    private static final Random random = new Random();
```

Comment:
```suggestion
    private final Random random = new Random();
```
think of it like a dependency injeciton

Reply:
Not fixed, remove static